pkg/util/testing/not_found_error_cmp.go

```diff
@@ -47,3 +47,27 @@ func (matcher *notFoundErrorMatch) FailureMessage(actual interface{}) (message s
 func (matcher *notFoundErrorMatch) NegatedFailureMessage(actual interface{}) (message string) {
 	return format.Message(actual, "not to be NotFound error")
 }
+
+func IsForbiddenError() types.GomegaMatcher {
```

Comment:
```suggestion
func BeForbiddenError() types.GomegaMatcher {
```

to match ginkgo's language